docs/moderator-handbook.md

```diff
@@ -190,7 +190,11 @@ Again, if you have any problems or questions, make a post with your actions in t
 
 ### Dealing with Posted Solutions
 
-When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+When a user replies in a help thread specific to the freeCodeCamp curriculum with a solution, remove it and either, kindly let them know that solutions are not permitted in help threads in your own words or, use the **Solution Instead of Help** "Canned reply".
```

Comment:
```suggestion
If a user replies in a help thread for the freeCodeCamp curriculum with a solution, remove it and use the **Solution Instead of Help** canned reply (or a similar response in your own words).
```